http/src/routing.rs

```diff
@@ -211,6 +213,7 @@ impl FromStr for Path {
             ["guilds", id, "regions"] => GuildsIdRegions(id.parse()?),
             ["guilds", id, "roles"] => GuildsIdRoles(id.parse()?),
             ["guilds", id, "roles", _] => GuildsIdRolesId(id.parse()?),
+            ["guilds", id, "templates"] => GuildsIdTemplates(id.parse()?),
```

Comment:
We talked about this this morning about a `"templates", _` case so I'll just leave a note here